src/api/controllers/password.reset.js

```diff
@@ -0,0 +1,110 @@
+import { hashSync, genSaltSync } from 'bcrypt';
+import models from '../models/index';
+import mailer from '../../helpers/Mailer';
+import environnement from '../../configs/environments';
+
+const { User } = models;
+const env = environnement.currentEnv;
+
+/**
+ * containing all user's model controllers (signup, login)
+ *
+ * @export
+ * @class Auth
+ */
+export default class PasswordReset {
+  /**
+   * A controller to send an email to a user
+   * requesting password recovery
+   *
+   * @param {Object} req - the request object
+   * @param {Object} res - the result object
+   * @returns {Boolean} true
+   */
+  static async sendRecoveryEmail(req, res) {
+    // Initialising variables
+    const result = {};
+    const status = 200;
+    const userEmail = req.body.email;
```

Comment:
oh.. I have a middleware to verify if the email provided in the body exists in the db. 
Although it will never reach the controller if the email doesn't exist in the db I will use the one from ```req.user``` instead